src/components/create-exam/step2.tsx

```
@@ -206,7 +207,7 @@ export const Step2 = ({ onBack }: Step2Props) => {
               return (
                 <FormItem>
                   <p className="text-lg font-bold text-brand-primary-950 mb-2">3. About quiz</p>
-                  <FormLabel>Any guidelines or final remarks you'd like to include?”</FormLabel>
+                  <FormLabel>Any guidelines or final remarks you'd like to include?</FormLabel>
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)